template-parts/header-post.php

```diff
@@ -0,0 +1,26 @@
+<?php
+$obj         = ucfwp_get_queried_object();
+$author      = get_the_author_meta( 'display_name', get_post_field( 'post_author', $obj->ID ) );
+$exclude_nav = get_field( 'page_header_exclude_nav', $obj );
+?>
+
+<?php if ( ! $exclude_nav ) { echo ucfwp_get_nav_markup( false ); } ?>
+
+<div class="container">
+	<h1 class="mt-3 mb-2"><?php echo $obj->post_title; ?></h1>
+	<div class="row mb-4 pt-1">
+		<div class="col-5 my-1">
+			<?php
+			if( $categories_obj = get_categories() ) {
+				foreach ($categories_obj as $each) {
+					?><span class="small my-3 mr-2 p-1 bg-faded"><?php echo $each->name; ?></span><?php
+				}
+			}
+			?>
```

Comment:
Should users be able to click on these categories to view other posts with these categories? Are we planning on creating a template for lists of posts within a category? These look a little bit like they could be clicked on, so we might want to adjust the design a little bit if they're just informational.

Reply:
Not for now, we just want to add the category in simple text, later on I can add the category page including the related posts.